packages/ssh-pool/src/commands/cd.js

```diff
@@ -3,5 +3,10 @@ import { joinCommandArgs, requireArgs } from './util'
 export function formatCdCommand({ folder }) {
   requireArgs(['folder'], { folder }, 'cd')
   const args = ['cd', folder]
+  const isWin = /^win/.test(process.platform)
```

Comment:
It is a constant, can be moved outside the function.

Reply:
done, i had fix it